mobify/lockup
# Updating deckard to 1.0.0

Updating deckard to 1.0.0

Status: **Ready to merge**

Reviewers: @kpeatt 
## Changes
- updated deckard in bower.json
## How to test-drive this PR
- checkout pinny `update-deckard` branch
- delete deckard
- `bower cache clean && bower install`
- ensure the examples still work
